src/components/component.ts

```diff
@@ -67,6 +70,36 @@ module Plottable {
         this._setup();
       }
       this._isAnchored = true;
+      this._onAnchorCallbacks.callCallbacks(this);
+      return this;
+    }
+
+    /**
+     * Adds a callback to be called on anchoring the current Component to the DOM.
+     * If the component is already anchored, the callback is called immediately.
+     *
+     * @param {AnchorCallback} callback The callback to be called on Anchor
+     *
+     * @return {Component}
+     */
+    public onAnchor(callback: AnchorCallback) {
+      if (this._isAnchored) {
+        callback(this);
+      }
+      this._onAnchorCallbacks.add(callback);
+      return this;
+    }
+
+    /**
+     * Removes a callback to be called on anchoring the Component to the DOM.
```

Comment:
Let's match the on/offAnchor docs exactly.

`onAnchor` has the word current.  Let's remove that